lib/compiler-llvm/src/abi/aarch64_systemv.rs

```diff
@@ -140,8 +140,7 @@ impl Abi for Aarch64SystemV {
                         Type::I32 | Type::F32 => 32,
                         Type::I64 | Type::F64 => 64,
                         Type::V128 => 128,
-                        Type::ExternRef => unimplemented!("externref in the llvm backend"),
-                        Type::FuncRef => unimplemented!("funcref in the llvm backend"),
+                        Type::ExternRef | Type::FuncRef => 64, /* pointer */
```

Comment:
The presence of VMOffsets is optional in this function. The issue is that we don't have a VMContext when producing trampolines, so we don't have a VMOffset for it either.

Reply:
Yeah, it's fine. Really not a big deal because we do this thing in other places in compiler-llvm where we don't use data from `VMOffsets` so things may get out of sync. I think it's fine to not worry about it for now and we can update compiler-llvm later if we decide that that's a high enough priority thing to do.

Reply:
Really, a lot of `VMOffsets` stuff is static. In this case I don't know if we actually need a `VMOffsets` but, yeah, generally speaking getting the size of things doesn't require anything other than knowing the size of a pointer on the target architecture. Maybe that means it needs to be split into two pieces